dashboard/views.py

```diff
@@ -28,14 +29,23 @@ def home(request):
     :return: The home page template or the login template.
     """
     if request.user.is_authenticated and is_integrated(request.user):
-        #  If not already present, store the experiment start date after the first login
+        #  Store the experiment start date after the first login and redirect to questionnaire
         if not StartDate.objects.filter(user=request.user).exists():
             StartDate(user=request.user).save()
+            # redirect to self-efficacy questionnaire
 
-        # Retrieve and parse user PA data per week (for the 2-week steam graph)
-        # start date as datetime object
+        # Retrieve user start date as datetime object and compute end_date
         start_date = StartDate.objects.get(user=request.user).start_date
+        end_date = start_date + timedelta(days=15)
+        print("""
+              start_date = {start_date}
+              today      = {date.today()}
+              end_date   = {end_date}
+              """)
+        # if date.today() == end_date:
+        # redirect to self-efficacy questionnaire
```

Comment:
This is the second entry point of the questionnaire.
We must hit it after two weeks of experiment.